### DISCUSSION TITLE
Everytime Brave updates, I have to reimport the cert

### DISCUSSION THREAD

**Discussion by iwyatt** [coddingtonbear/obsidian-web#67] [Exotic Browser Support]
![image](https://user-images.githubusercontent.com/7514713/202040748-377d25a0-4ac7-4bd1-87f2-019a3bdf4c91.png)

I get this error message after every update to Brave and  have to reimport the certification.

This might be user error because I don't usually import certs.
It also might be an issue with Brave using an Extension designed for Chrome.

Otherwise, curious if this is a known limitation or bug?

Any help or insight would be appreciated.

Thank you!

Edit: It might be that I need to import under this heading instead of "personal", though I get some "Scary" warnings when attempting to do so: 

![image](https://user-images.githubusercontent.com/7514713/202042075-bcae1adb-bfc5-4c9e-9c51-f8caf3915a4a.png)

Can anyone confirm?

**Comment by coddingtonbear** [comment]
Couldn't say!  Unfortunately, this extension is developed for and tested only on Chrome.  I wish I could help more, but it's already difficult enough just finding the time to support a single browser at the moment.  Perhaps others might have ideas for you, though?

**Reply by iwyatt** [reply]
Totally understandable. Thanks for looking at least!
